phoenix-core/src/main/java/org/apache/phoenix/index/IndexMaintainer.java

```diff
@@ -1887,6 +1899,11 @@ public boolean isImmutableRows() {
         return immutableRows;
     }
     
+    public boolean isTableIndex() {
```

Comment:
I haven't come across any think as table index and might confuse while reading as global index comes with its own table. How about isViewIndex() with a negation?

Reply:
Makes sense, made the change and pushed in latest commit